pkg/apis/jvmbuildservice/v1alpha1/dependencybuild_types.go

```diff
@@ -66,7 +66,10 @@ type DependencyBuildList struct {
 }
 
 type BuildRecipe struct {
+	//deprecated task string
```

Comment:
will remove this field when we come back and remove the maven/gradlecluster task files